libraries/chain/block_log.cpp

```diff
@@ -1516,6 +1529,16 @@ namespace eosio { namespace chain {
           } , *context);
    }
 
+   // static
```

Comment:
You can add a comment saying returning `0` if block logs do not exist. This comment is optional.

Reply:
Since there are no comments on any of the methods, including `first_block_num()`. I think I'll leave it as is to avoid having to wait on another ci/cd run.

Reply:
Just leave as is.